src/BDF.jl

```diff
@@ -737,7 +737,7 @@ function splitBDFAtTrigger(fname::AbstractString, trigger::Integer; from::Real=0
     stopPoints =  [sepPoints; size(data)[2]] 
     
     for i=1:nChunks
-        thisFname = string(split(fname, ".")[1], "_", i, ".", split(fname, ".")[2])
+        thisFname = joinpath(dirname(fname), basename(fname)[1:end-4] * "_" * string(i) * basename(fname)[end-3:end])
```

Comment:
Could probably have just done 
```julia
joinpath(fname[1:end-4] * "_" * string(i) * basename(fname)[end-3:end])
```
I was hoping there was an `extension(fname)` function then it could have been 

```julia
joinpath(dirname(fname), basename(fname)[1:end-4] * "_" * string(i) * extension(fname))
```